docs/examples/soroban/invoke_contract_functions.md

```diff
@@ -133,19 +134,15 @@ source_account
 |> Stellar.TxBuild.envelope()
 ```
 
-### With Stellar Account Authorization (WIP: Preview 10 support)
+### With Stellar Account Authorization
```

Comment:
IMO we should add an explanatory text about this use case, we can add it to every title as well.

For example, this use case is when the tx submitter is different from invoker.